src/pydisagg/ihme/splitter.py

```diff
@@ -101,6 +92,234 @@ def remove_prefix(self) -> None:
             setattr(self, field, getattr(self, field).removeprefix(self.prefix))
 
 
+class SexPopulationConfig(BaseModel):
+    index: list[str]
+    sex: str
```

Comment:
Here need more thinking, how can we distinguish which is "female" and which is "male", we need it to match and merge.

Reply:
Could we enforce that the user must use 1 and 2? or perhaps include it in the config?

Reply:
Yes, that can work, but let's have a discussion with everyone before we implement anything :)